src/services/pscVerificationService.ts

```diff
@@ -21,6 +22,17 @@ export const createPscVerification = async (request: Request, transaction: Trans
     if (!sdkResponse) {
         throw createAndLogError(`${createPscVerification.name} - PSC Verification POST request returned no response for transaction ${transaction.id}`);
     }
+
+    if (sdkResponse.httpStatusCode === HttpStatusCode.InternalServerError) {
+        const apiErrorResponseResource = sdkResponse as ApiErrorResponse;
+        const apiErrorResponse = apiErrorResponseResource.errors?.[0] as ApiErrorResponse;
+        const error = apiErrorResponse.errors?.[0].errorValues?.error as string;
+
+        if (error.match(Responses.PROBLEM_WITH_PSC_DATA as string)) {
```

Comment:
I see a Sonar issue for this condition and it suggests:
`RegExp(Responses.PROBLEM_WITH_PSC_DATA as string).exec(error)`